1. CWE Base Mod/CWE stable/localization/english/0_institutions_l_english.yml

```diff
@@ -2,3 +2,9 @@
     
     institution_gulag:0 "Concentration Camp Network"
     institution_home_affairs:0 "Secret Police"
+    institution_hos_office:0 "Head of State's Office"
+    institution_administration_service:0 "Federal Administration Services"
+    institution_elections_supervision:0 "Elections Watchdog"
+    institution_civil_service:0 "Civil Service"
+    institution_diplomatic_corps:0 "Diplomatic Corps"
+    institution_superstate:0 "Superstate Institutions"
```

Comment:
Might be good to put this in a separate localisation file

Reply:
Commit 2ba9aef solves this issue.